project/Dependencies.scala

```diff
@@ -33,7 +33,15 @@ object Dependencies {
 
     val r2dbcSpi = "io.r2dbc" % "r2dbc-spi" % "0.9.1.RELEASE"
     val r2dbcPool = "io.r2dbc" % "r2dbc-pool" % "0.9.2.RELEASE"
-    val r2dbcPostgres = "org.postgresql" % "r2dbc-postgresql" % "0.9.3.RELEASE"
+
+    // This is here because sbt's ivy resolver doesn't properly support packaging.type when
+    // resolving via sbt-license-report, see https://github.com/sbt/sbt-license-report/issues/87
+    val r2dbcPostgres = Seq(
```

Comment:
Here is an output of the dependency tree without the overrides which you can use as a confirmation that the versions line up

```
[info]   |   +-org.postgresql:r2dbc-postgresql:0.9.3.RELEASE
[info]   |     +-com.ongres.scram:client:2.1
[info]   |     | +-com.ongres.scram:common:2.1
[info]   |     |   +-com.ongres.stringprep:saslprep:1.1
[info]   |     |     +-com.ongres.stringprep:stringprep:1.1
[info]   |     |     
[info]   |     +-io.projectreactor.netty:reactor-netty:1.0.24
[info]   |     | +-io.projectreactor.netty.incubator:reactor-netty-incubator-quic:0.0..
[info]   |     | | +-io.netty.incubator:netty-incubator-codec-native-quic:0.0.33.Final
```